rampwf/utils/tests/test_testing.py

```diff
@@ -0,0 +1,22 @@
+import shutil
+from tempfile import mkdtemp
+from rampwf.kits import fetch_ramp_kit
+from rampwf.utils import assert_submission
+
+# later on we need to make a util to list kits to avoid repeating the same
+# list everywhere
+RAMP_KITS = ('boston_housing',
```

Comment:
The purpose of having a common library for the whole framework is IMO to have the starting kits pass the submission tests at anytime.

@kegl what is your opinion on this ?